packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -592,14 +592,26 @@ void main() {
       );
     });
 
-    testWidgets('width of wheel in background does not increase at large widths', (WidgetTester tester) async {
+    testWidgets('wheel does not bend outwards', (WidgetTester tester) async {
```

Comment:
This test also didn't work properly because of the bug https://github.com/flutter/flutter/issues/121343

Values were:
1
decemberX = 250
octoberX = 250

2
decemberX = 218.5
octoberX = 218.5

So the distance in both cases was zero

In actuality these values were

262.6779112246874
250.75842555233146
distance 11.919485672355961

232.8608197943362
219.35910151060554
distance = 13.501718283730668